Slavahaim/hometask-6/spec/my_hash_spec.rb

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/env ruby
+# frozen_string_literal: true
+
+require '../my_hash'
+require 'rspec'
+
+# rubocop:disable Metrics/BlockLength
+RSpec.describe MyHash do
+  let(:my_hash) { described_class.new }
+
+  describe '#[]=' do
```

Comment:
For this case we should more contexts  `when a key is missed`, could you wrapp your  it 'return changed hash size and write key=>value' into context